week-1/K-functions-parameters/exercise.js

```diff
@@ -1,6 +1,6 @@
 // Complete the function so that it takes input parameters
-function multiply() {
-  // Calculate the result of the function and return it
+function multiply(a, b) {
+  return 3 * 4;
```

Comment:
Try returning the parameters of the function instead of 3 * 4 and try the function with different numbers. Currently this will only return 3 * 4

Reply:
Updated parameters to multiply any given number. Thanks, Alexander!